configuration/outbound.md

```diff
@@ -0,0 +1,160 @@
+# Outbound push [beta]
+
+*Outbound is only available in 3.5.0 and above*
```

Comment:
Maybe specify Medic WebApp 3.5.0. One day Medic-conf might be in v3 and this would be quite confusing as it is a configuraiton document